ods_ci/tests/Tests/400__ods_dashboard/420__model_serving/421__model_serving_gpu.robot

```diff
@@ -50,6 +51,9 @@ Verify GPU Model Deployment Via UI
     Should Be Equal As Strings    ${type}    "g4dn.xlarge"
     Verify Model Status    ${MODEL_NAME}    success
     Set Suite Variable    ${MODEL_CREATED}    True
+    [Teardown]    Run Keyword If Test Failed    Get Events And Pod Logs    namespace=${PRJ_TITLE}
+    ...    label_selector=name=modelmesh-serving-${runtime_pod_name}
+
```

Comment:
Same comment on the runtime pod name, also too many empty lines between the two tests?